zebra-consensus/src/checkpoint.rs

```diff
@@ -818,6 +822,10 @@ mod tests {
     }
 
     #[tokio::test]
+    async fn multi_item_checkpoint_list_test() -> Result<(), Report> {
```

Comment:
I had to split these tests into inner and outer functions to separate the `#[tokio::test]` and `#[spandoc]` attributes which do not play together well due to bugs in span preservation in proc macros. If you put both attributes on the same function all "unused doc comment" warnings will be reported at `lib.rs:1` so its really difficult to figure out why the warning got introduced.